srcalc/src/main/java/gov/va/med/srcalc/web/view/admin/NumericalRangeBuilder.java

```diff
@@ -0,0 +1,152 @@
+package gov.va.med.srcalc.web.view.admin;
+
+import com.google.common.base.MoreObjects;
+
+import gov.va.med.srcalc.domain.model.NumericalRange;
+
+/**
+ * <p>Builds a {@link NumericalRange} from incrementally-specified component
+ * parts. This class is useful because NumericalRange is immutable: you must
+ * specify all properties in its constructor and you can't use it as a Java
+ * bean.</p>
+ * 
+ * <p>Per Effective Java Item 17, this class is marked final because it was not
+ * designed for inheritance.</p>
+ */
+public final class NumericalRangeBuilder
+{
+    private float fLowerBound;
+    private boolean fLowerInclusive;
+    private float fUpperBound;
+    private boolean fUpperInclusive;
+    
+    /**
+     * Constructs an instance. The default bounds are [0,100.0].
+     */
+    public NumericalRangeBuilder()
+    {
+        fLowerBound = 0.0f;
+        fLowerInclusive = true;
+        fUpperBound = 100.0f;
```

Comment:
I want the default value on the UI to be 100. (Because 200 is too arbitrary.) I will just change the default for AbstractNumericalVariable to 100.